### ISSUE TITLE
Warning: Undefined array key "source_id"

### ISSUE THREAD

**Issue by MatzeKitt** [Automattic/wordpress-activitypub#902]
### Quick summary

My Sentry logged an error when requesting `/comments/feed/` or POSTing to `/wp-comments-post.php`.

### Steps to reproduce

🤷🏻‍♂️

### What you expected to happen

No error 🙃

### What actually happened

Error:

```
ErrorException: Warning: Trying to access array offset on value of type null
#15 /wp-content/plugins/activitypub/includes/class-comment.php(376): Activitypub\Comment::get_source_url
#14 /wp-content/plugins/activitypub/includes/class-comment.php(396): Activitypub\Comment::remote_comment_link
#13 /wp-includes/class-wp-hook.php(326): WP_Hook::apply_filters
#12 /wp-includes/plugin.php(205): apply_filters
#11 /wp-includes/comment-template.php(860): get_comment_link
#10 /wp-includes/feed.php(319): comment_link
#9 /wp-includes/feed-rss2-comments.php(96): require_once
#8 /wp-includes/template.php(810): load_template
#7 /wp-includes/functions.php(1670): do_feed_rss2
#6 /wp-includes/class-wp-hook.php(326): WP_Hook::apply_filters
#5 /wp-includes/class-wp-hook.php(348): WP_Hook::do_action
#4 /wp-includes/plugin.php(517): do_action
#3 /wp-includes/functions.php(1634): do_feed
#2 /wp-includes/template-loader.php(48): require_once
#1 /wp-blog-header.php(19): require
#0 /index.php(17): null
```

### Impact

One

### Available workarounds?

No but the platform is still usable

**Comment by pfefferle** [comment]
this should be already fixed in 3.3.1!

...but thanks for reporting ☺️

**Comment by MatzeKitt** [comment]
Oh yeah, didn’t update yet. 😬
